### PR TITLE
Update database-connector.md

### PR DESCRIPTION
**Pull request by grootjans**

### CODE REVIEW

**Comment by grootjans** [comment]
@passalaqua please review!

**Comment by passalaqua** [comment]
Looks good to me!

File: content/appstore/connectors/database-connector.md [hunk 1]
```diff
@@ -77,6 +77,10 @@ These are the results of the actions:
 * [Presto](https://prestodb.github.io/docs/current/installation/jdbc.html)
 * [SQLite](https://bitbucket.org/xerial/sqlite-jdbc/downloads/)
 
+{{% alert type="info" %}}
+If you intend to connect to Sql Server using integrated security, please be aware that the JDBC driver in the `userlib` folder of the module needs to match the version supplied with the Mendix platform.
```
Comment (basdebakker) [inline content/appstore/connectors/database-connector.md:81]: The SQL in SQL Server is written in capitals. Also, there isn't really a userlib folder of the module, only one for the project as a whole.

Comment (grootjans) [reply content/appstore/connectors/database-connector.md]: updated.

Comment (basdebakker) [reply content/appstore/connectors/database-connector.md]: Looks good to me, too.

**Review by Adam-Dupaski — APPROVED** [review]

**Pull request merged**